### PR TITLE
[17.0][MIG] product_category_active: Migration to 17.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 17.0

Please @pilarvargas-tecnativa and @carlos-lopez-tecnativa can you review it?

@Tecnativa TT52280

### CODE REVIEW

File: product_category_active/readme/ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- when you try to deactivate a category from the tree view by clicking
+  on the boolean_toggle widget, if a validation error pop-up appears
+  indicating that the element cannot be deactivated, then the element is
+  NOT deactivated, although the corresponding line turns gray as if it
+  had been disabled, until that tree view is refreshed.
```
Comment (carlos-lopez-tecnativa) [inline product_category_active/readme/ROADMAP.md]: I believe this issue is now resolved. Could you review it and update the roadmap if necessary?

image

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM, Just a minor change to the roadmap, if applicable

**Review by pilarvargas-tecnativa — APPROVED** [review]

**Pull request merged**